chia/util/condition_tools.py

```diff
@@ -111,6 +111,7 @@ def coin_announcements_for_conditions_dict(
     output_announcements: Set[Announcement] = set()
     for cvp in conditions_dict.get(ConditionOpcode.CREATE_COIN_ANNOUNCEMENT, []):
         message = cvp.vars[0]
+        assert len(message) <= 1024
```

Comment:
We are okay with `assert` for now, but it should change to a real `raise` at some point (and maybe with its own error).

Reply:
I put a `TODO` comment in.